packages/contracts/src/contracts/current/protocol/Exchange/MixinExchangeCore.sol

```diff
@@ -41,7 +41,7 @@ contract MixinExchangeCore is
 {
     // Mappings of orderHash => amounts of takerTokenAmount filled or cancelled.
     mapping (bytes32 => uint256) public filled;
-    mapping (bytes32 => uint256) public cancelled;
+    mapping (bytes32 => bool) public cancelled;
```

Comment:
I can update [PR428](https://github.com/0xProject/0x-monorepo/pull/428) to optimize this into a single var.